python/llm/example/CPU/vLLM-Serving/README.md

```diff
@@ -0,0 +1,75 @@
+# Serving LLAMA models using vLLM on Intel platforms (experimental support)
+
+This example demonstrates how to serving a llama2-7b model using BigDL-LLM 4 bits optimizations with xeon CPUs with adapted vLLM.
+
+
+## Example: Serving llama2-7b using Xeon CPU
+
+In this example, we will run Llama2-7b model using 48 cores in one socket and provide `OpenAI-compatible` interface for users.
+
+### 1. Install
+
+The original [vLLM](https://github.com/analytics-zoo/vllm) is designed to run with `CUDA` environment. To adapt vLLM into `Intel` platforms, install the dependencies like this:
```

Comment:
wrong link? 
please declare that the vllm example and code are ported from https://github.com/vllm-project/vllm

Reply:
Fixed